.github/workflows/publishPreRelease.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install dependencies, check for dependency version consistency, build all packages in the repo, and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: iTwin Viewer Components React NPM Publish
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PUBLISH_PAT }}
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: |
+        set -e
+        echo 'Installing...'
+        node common/scripts/install-run-rush.js install
+        echo 'Building...'
+        node common/scripts/install-run-rush.js rebuild
+        echo 'Publishing...'
+        git config user.name "iTwinVCRWorkflow"
+        git config user.email "itwinviewer@users.noreply.github.com"
```

Comment:
@calebmshafer is there a preferred git user to use?

Reply:
good point, let me try and switch that. It may be the cause of our publishing issues.

Reply:
Actually, I wouldn't have a separate workflow for this. I'd try to do it in the same one.